src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+
+  const cssDeclarations = sourceString.split(';');
+
+  const cssObject = {};
+
+  cssDeclarations.forEach((cssDeclaration) => {
+    const trimmedcssDeclaration = cssDeclaration.trim();
+
+    if (trimmedcssDeclaration) {
+      const parts = trimmedcssDeclaration.split(':');
+
+      const property = parts[0].trim();
+
+      const value = parts[1].trim();
+
+      cssObject[property] = value;
+    }
```

Comment:
This part of the code can be greatly improved.
Let`s do it: 

```suggestion
// Check if both parts exist after the split
      if (parts.length === 2) {
//Using destructuring to trim key and value simultaneously.
        const [property, value] = parts.map(part => part.trim());
        
        cssObject[property] = value;
      }
    }
```